packages/web-api/src/types/request/chat.ts

```diff
@@ -0,0 +1,224 @@
+import type { KnownBlock, Block, MessageAttachment, LinkUnfurls, MessageMetadata } from '@slack/types';
```

Comment:
I am still not 100% confident that all the patterns here are correct ... but well done!

Reply:
Yes I think this is a risky area.. all possible combinations of Block elements, etc, may be hard to verify... but I think as long as we are OK to tell developers that find a problem, "please use `// @ts-expect-error` as a workaround until we release a patch", it should be OK.